test/perf.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+cd "$(dirname "$0")"
+
+TIME=${2:-10}
+
+create() {
+    limactl create --name server --tty=false lima.yaml &
+    limactl create --name client --tty=false lima.yaml &
+    wait
+}
+
+setup() {
+    limactl start server &
+    limactl start client &
+    wait
+    nohup limactl shell server iperf3 --server --daemon
+}
+
+run() {
+    server_address=$(limactl shell server ip -j -4 addr show dev lima0 | jq -r '.[0].addr_info[0].local')
+    limactl shell client iperf3 --client $server_address --length 1m --time $TIME
+}
```

Comment:
Can be useful to have host-to-vm and vm-to-vm tests. Also it is useful to test one vm and 2 vms separately, since performance drop which every vm running, even if we don't access it in the tests due to flooding (#58).

When testing I'm usually start and stop the client vm manually with limactl. It can be nice to automated the entire test.